tools/src/main/scala/org/apache/spark/sql/rapids/tool/FilterAppInfo.scala

```diff
@@ -39,15 +41,27 @@ class FilterAppInfo(
       event.time,
       event.sparkUser
     )
-    appInfo = Some(thisAppInfo)
+    appStartInfo = Some(thisAppInfo)
+  }
+
+  def doSparkListenerEnvironmentUpdate(event: SparkListenerEnvironmentUpdate): Unit = {
+    logDebug("Processing event: " + event.getClass)
+    val sparkProperties = event.environmentDetails("Spark Properties").toMap
+    configInfo = Some(EnvironmentInfo(sparkProperties))
```

Comment:
still question?

Reply:
since we have changed it to spark-property now I didn't specify that we don't support hadoopProperties/system Properties in the doc. Please let me know if that should be explicitly mentioned.